src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -4089,4 +4089,7 @@ To make this file into a new template, save it to a different folder, then move
   <data name="PackageIncompatibleFilterTooltip" xml:space="preserve">
     <value>Versions maybe incompatible with the current setup</value>
   </data>
+  <data name="PackageVersionTooltip" xml:space="preserve">
+    <value>Versions marked with .x include all versions in the specified major release</value>
```

Comment:
In future when we start supporting asterisk with minor version 3.2.*, the tooltip will become incorrect as it will always indicate major release. @hwahlstrom @avidit . Thoughts?

Reply:
@zeusongit Update the tooltip when we start supporting minor version?

Reply:
We won't be able to update it in 3.4, right?

Reply:
@zeusongit Will supporting minor version be needed for 3.4 then? Should I go ahead with the cherrypick to 3.4?
FYI @QilongTang

Reply:
Got it, we should make a call on that then before merging to 3.4 branch. @hwahlstrom

Reply:
Changed in here https://github.com/DynamoDS/Dynamo/pull/15648